src/website/src/pages/ec/getting-started/index.mdx

```diff
@@ -19,7 +19,52 @@ New components are continuously being added to the library. The team is also con
 
 ## How to use ECL
 
-Here is an example of default HTML file using ECL:
+ECL distributes its resources in different ways and depending on the implementation approach, things might change a bit, generally speaking, given the HTML markup defined by ECL, applying the distrbuted css and javascript on top of it would ensure the expected look and feel.
+
+### Where to get the ECL resources?
+
+**ECL is distributed primarily as npm packages and this is the recommended way of getting it**.
+When used in a node project this is surely the easiest and more reliable way, the new versions can be automatically detected and fetched and the ECL resources could be even rebuilt on the client side by fetching all the needed packages one by one.
+
+In every release, starting from 3.9.1, a file containing a list of all the available packages on npm is deployed on the github release page, this list is quite long and you don't surely need, in a normal situation, to know all the available packages as you don't need to fetch them all.
+
+#### Identifying the packages
+
+ECL releases npm packages for:
+
+- builder (used to compile css and js) `@ecl/builder`
+- presets (EC and EU) `@ecl/preset-ec`
+- vanilla components (css and js) `@ecl/vanilla-component-{name}`
+- twig components (twig template) `@ecl/twig-component-{name}`
+- spec packages (data used in the demos) `@ecl/specs-component-{name}`
+- resources packages `@ecl/resources-ec-{name}`
+
+#### Presets
+
+The main ECL package that is usually needed in any environment where ECL is going to be used:
+
+- `preset-ec`
+
+In this package all the css, js and resources like icons and logos are present, there is no need, then, to look for other packages holding part of these resources.
+
+`npm install @ecl/preset-ec`
+OR
+`yarn add @ecl/preset-ec`
+
+### Alternative approaches:
+
+ECL is also distributing its presets on a CDN, all the files distributed in the package mentioned before are also available in the CDN, this is the pattern to reach them:
+`https://cdn{1,2,3 or 4}.fpfis.tech.ec.europa.eu/ecl/{tag}/{system}/{path/to/the/asset}`
+
+Ex: https://cdn1.fpfis.tech.ec.europa.eu/ecl/v3.9.1/ec/styles/optional/ecl-ec-default.css
+
+These resources can be used loading them directly from the CDN, with the limitation that it's not possible to serve svgs from an external domain without using some workaround, see the "Additional information" below.
+
+For different reasons, starting from performance and network traffic, **the recommended way of using ECL is to fetch its resources for then hosting them "locally"**.
+
+In order to ease the access to the package containing them, the main package mentioned before: `preset-ec` is also deployed in the github release page and can be downloaded from there.
+
+Here is an example of a default HTML file using ECL:
```

Comment:
no indeed, that is the reason why the example is at the end, because that is the (unique) way to use ECL once you have decided whether to fetch the resources or use the CDN.